### PR TITLE
RHIF-190: rename Overview to Dashboard, Inventory to Systems navs in stage env

### PR DESCRIPTION
**Pull request by mkholjuraev**
This renames the following navigations in STAGE env to be aligned with the new Navigation changes [RHIF-190](https://issues.redhat.com/browse/RHIF-190).

Overview => Dashboard (navigation id is also changed to dashboard to make it same is the app URL)
Inventory => Systems

I will create the same PR for prod env once gets verified and ack.

### CODE REVIEW

**Comment by mkholjuraev** [comment]
@Hyperkid123 A quick is appreciated for this important PR.

File: static/stable/stage/navigation/rhel-navigation.json [hunk 1]
```diff
@@ -3,9 +3,9 @@
     "title": "Red Hat Insights",
     "navItems": [
       {
-        "id": "overview",
+        "id": "dashboard",
```
Comment (Hyperkid123) [inline static/stable/stage/navigation/rhel-navigation.json:6]: This changes has to be reflected in `service.json` if the link is supposed to show up in the new dropdown

Comment (mkholjuraev) [reply static/stable/stage/navigation/rhel-navigation.json:6]: Thank you! Do you mean all changes or only the change of nav id?

Comment (mkholjuraev) [reply static/stable/stage/navigation/rhel-navigation.json:6]: Went through service.json under /stage/services folder. 

1. As the dashboard is not a real service but an overall page for RHEL insights, it seems we do not have to make any changes for Overview => dashboard change 
2. The inventory is renamed already inside the dropdown without changing it's ID. This also seems not needed to be reflected in /stage/services/servises.json file.

Let me know if I miss anything, it is my first PR in this new repo :)

**Review by Hyperkid123 — APPROVED** [review]

**Pull request merged**